.github/workflows/antsibull-build.yml

```diff
@@ -28,12 +28,12 @@ jobs:
             options: '-e antsibull_ansible_version=8.99.0'
             python: '3.9'
             antsibull_changelog_ref: 0.14.0
-            antsibull_core_ref: main
+            antsibull_core_ref: stable-2
           - name: Ansible 9
             options: '-e antsibull_ansible_version=9.99.0'
             python: '3.11'
             antsibull_changelog_ref: main
-            antsibull_core_ref: main
+            antsibull_core_ref: stable-2
           - name: Ansible 10
             options: '-e antsibull_ansible_version=10.99.0'
             python: '3.12'
```

Comment:
Shouldn't the `antsibull_core_ref` below also be changed to `stable-2` so we can do breaking changes in antsibull-core without disturbing CI in ansibull?